publications-2022.bib

```diff
@@ -812,6 +812,21 @@ @PhDThesis{2022:mang:phase-field
   doi     = {10.15488/11928}
 }
 
+@Article{2022:mohebujjaman:high,
+  author  = {Muhammad Mohebujjaman},
+  title   = {High Order Efficient Algorithm for Computation of {MHD} Flow Ensembles},
+  journal = {Advances in Applied Mathematics and Mechanics},
+  year    = 2022,
+  volume  = 14,
+  number  = 5,
+  pages   = {1111--1137},
+  month   = jun,
+  doi     = {10.4208/aamm.oa-2021-0154},
+  url     = {https://global-sci.org/intro/article_detail/aamm/20554.html},
+  publisher = {Global Science Press},
+  preprint = {https://arxiv.org/abs/2101.06852}
+}
```

Comment:
This article uses freefem, whereas an earlier version on researchgate uses deal.II. I will adjust it.